analyses/independent-samples/02-generate-indpependent-rnaseq.R

```diff
@@ -0,0 +1,100 @@
+# 02-generate-independent-rnaseq.R
```

Comment:
Looks like you have a little typo in the file name (this line is correct  😉).

Reply:
Oh no.. I'll rename the file